requirements.txt

```diff
@@ -0,0 +1,13 @@
+click==8.1.3
```

Comment:
Regenerate the requirements.txt file using PyCharm editor tools, then it will only contain the packages you are installing, not their dependency packages.

Reply:
Let it be there.